### PR TITLE
Disable form submission on fractal site.

### LINKED ISSUE #2101: disable form submission on components site
request from GSA IT. See slack:

https://gsa-tts.slack.com/archives/C050HRGN7/p1502394104797479

### PR DESCRIPTION
**Pull request by toolness**
Fixes #2101.

### CODE REVIEW

File: src/components/_uswds.njk [hunk 1]
```diff
@@ -7,5 +7,13 @@
...
     {% block body %}{{ yield }}{% endblock %}
     <script src="{{ uswds.path }}/js/uswds.min.js"></script>
+    <script>
+    window.addEventListener('load', function() {
+      document.body.addEventListener('submit', function(e) {
+        // https://github.com/18F/web-design-standards/issues/2101
+        e.preventDefault();
+      }, true);
+    }, false);
+    </script>
```
Comment (toolness) [inline src/components/_uswds.njk]: I ended up using an inline script instead of creating a JS file because the static assets directory for our fractal site is `dist`, and I didn't want to pollute it with static files that are unrelated to core USWDS.

Comment (msecret) [inline src/components/_uswds.njk]: That makes sense.

**Comment by wslack** [comment]
Thank y'all! I put the site in for re-scan.
